packages/suite/src/hooks/suite/useMetadata.ts

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from './useSelector';
+
+export const useMetadata = () => {
```

Comment:
huh, right! let me do that. I am the oldschool boy..

Reply:
I did some changes here 3076e26a88a154fc0d82637fa9b4363bca4010a5. I still somehow have trouble to abandon hook completly because it nicely wraps multiple imports (useSelector and particular selectors). But I don't really understand recent trends so I will basically whatever you tell me to do

Reply:
it's just a nitpick, but you really don't need the hook here and you can use it on all places just like
```
 const isLabelingAvailable = useSelector(selectIsLabelingAvailable);
 ```

Reply:
changed